interface-definitions/service_dhcp-server.xml.in

```diff
@@ -63,6 +63,18 @@
                   <constraintErrorMessage>Invalid DHCP failover peer status</constraintErrorMessage>
                 </properties>
               </leafNode>
+              <leafNode name="split">
+                <properties>
+                  <help>The split statement specifies the split between the primary and secondary for the purposes of load balancing.</help>
+                  <valueHelp>
+                    <format>u32:0-256</format>
+                    <description>Split value</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 0-256"/>
+                  </constraint>
+                </properties>
```

Comment:
Any reason to not use `<defaultValue>128</defaultValue>`?

Reply:
As I understand In the "secondary" node shouldn't be any `split` option
That's why we cannot use default

Reply:
Yes I've seen this. We have a lot of other places where default values are only set if certain preconditions match. It's a bit more complicated but then we have a non disconnect between XML (help) and CLI/implementation.

Inspiration:
```
# git grep "del default_values"
src/conf_mode/pki.py:                    del default_values['certificate'][name]['acme']
src/conf_mode/protocols_ospf.py:        del default_values['default_information']
src/conf_mode/protocols_ospf.py:        del default_values['mpls_te']
src/conf_mode/protocols_ospf.py:        del default_values['graceful_restart']
src/conf_mode/protocols_ospf.py:            del default_values['area'][area_num]['area_type']['nssa']
src/conf_mode/protocols_ospf.py:            del default_values['redistribute'][protocol]
src/conf_mode/protocols_ospf.py:        del default_values['redistribute']
src/conf_mode/protocols_ospf.py:            del default_values['interface'][interface]['dead_interval']
src/conf_mode/protocols_ospfv3.py:        del default_values['default_information']
src/conf_mode/protocols_ospfv3.py:        del default_values['graceful_restart']
src/conf_mode/protocols_ospfv3.py:            del default_values['redistribute'][protocol]
src/conf_mode/protocols_ospfv3.py:        del default_values['redistribute']
src/conf_mode/protocols_pim.py:            del default_values['interface'][interface]['igmp']
src/conf_mode/service_https.py:        del default_values['api']
src/conf_mode/system_flow-accounting.py:            del default_values[flow_type]
```